sw/device/silicon_creator/rom_exts/rom_ext_boot_policy_ptrs.h

```diff
@@ -0,0 +1,57 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXTS_ROM_EXT_BOOT_POLICY_PTRS_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXTS_ROM_EXT_BOOT_POLICY_PTRS_H_
+
+#include "sw/device/silicon_creator/lib/manifest.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+static_assert((TOP_EARLGREY_EFLASH_SIZE_BYTES % 2) == 0,
+              "Flash size is not divisible by 2");
+
+#ifndef OT_OFF_TARGET_TEST
+/**
+ * Returns a pointer to the manifest of the first owner boot stage image stored
+ * in flash slot A.
+ *
+ * @return Pointer to the manifest of the first owner boot stage image in slot
+ * A.
+ */
+// TODO(#7893): Should these be volatile?
+inline const manifest_t *rom_ext_boot_policy_manifest_a_get(void) {
+  return (const manifest_t *)(TOP_EARLGREY_EFLASH_BASE_ADDR +
+                              MANIFEST_LENGTH_FIELD_MAX);
```

Comment:
I think it would be nice to have a more explicit name for the ROM_EXT fixed size definition, as this is something the Silicon Owners will have to be aware of in general.

Reply:
As we discussed minimum and maximum allowed lengths for ROM_EXT and first owner boot stage images will be different. So I added individual constants for both stages to `manifest.h` (using `define`s so that `bindgen` output is nicer) and moved length checks to mask ROM and ROM_EXT boot policy implementations. PTAL.